src/components/restaurant/business-home.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+// import {useSelector} from "react-redux";
+import {Link} from "react-router-dom";
+import "./business-home.css";
+import BusinessInfo from "./business-info";
+import BusinessPost from "./business-post";
+import FeaturedItems from "./featured-items";
+
+const BusinessHome = (
+{
+    restaurant = {
+        _id: "123",
+        name: "Red Lobster",
+        bannerPicture: "redlobster-banner.jpeg",
+        profilePicture: "redlobster-avatar.png",
+        handler: "@redlobster",
```

Comment:
I think we should change "handler" to "handle" - small detail for now but just for consistency when we create the database so we don't have to come back to refactor any of the property names

Reply:
Will do!